src/libraries/System.Private.CoreLib/src/System/IO/Strategies/FileStreamHelpers.Windows.cs

```diff
@@ -14,9 +14,8 @@ namespace System.IO.Strategies
     // this type defines a set of stateless FileStream/FileStreamStrategy helper methods
     internal static partial class FileStreamHelpers
     {
-        // Async completion/return codes shared by:
-        // - AsyncWindowsFileStreamStrategy.ValueTaskSource
-        // - Net5CompatFileStreamStrategy.CompletionSource
+        // Async completion/return codes used by
+        // SafeFileHandle.OverlappedValueTaskSource
         internal static class TaskSourceCodes
```

Comment:
This class is unused but I didn't remove it to avoid surprising any new code that might decide to use it in the meantime until the PR is merged.